views/openacademy_course_views.xml

```diff
@@ -28,6 +28,13 @@
             <search string="Courses">
                 <field name="name"/>
                 <field name="description"/>
+                <filter name="filter_responsible"
+                string="Responsible"
+                domain="[('responsible_id', 'ilike', self)]"/>
+                <group string="Group By">
+                    <filter name="group_by_responsible" string="Group by Responsible"
```

Comment:
This will be inside the "Group by" button, so text "Group by" would be redundant

Reply:
Not applied yet. I meant texts like "Group by Responsible" don't make sense inside the group by button, because all items are intended to group, so "group by" is redundant.

Reply:
Okay, I think I fixed it.

Reply:
No, the string is a mandatory attribute (AFAIK), don't remove it, just use `string="Responsible"`

Reply:
Done.